scripts/build.sh

```
@@ -2,27 +2,13 @@
 
 set -ex;
 
-VERSION=$(grep '^version:' pubspec.yaml | sed -e 's/version: //')
+export VERSION=$(grep '^version:' pubspec.yaml | sed -e 's/version: //')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)